README.md

```diff
@@ -10,8 +10,14 @@ Example twitter feed: [http://www.twitter.com/ericapets](http://www.twitter.com/
 
 ## Set Up for Local Development
 
+### System Requirements
```

Comment:
My preference is not to include gems in the readme and let the Gemfile serve that purpose. It eliminates the need to keep them in sync.